package-manager/pkg/packmgr/installer_test.go

```
@@ -222,11 +265,11 @@ func TestInstallRejectsInvalidSource(t *testing.T) {
 			os.Remove(filepath.Join(dir, "app.wasm"))
 		}},
 		{name: "entrypoint escapes source dir", mutate: func(dir string) {
-			manifest := []byte(`{"schema_version":"ailuo.package.v2","id":"demo.pkg","version":"1.0.0","components":[{"id":"core","mode":"hosted","entrypoint":"../outside"}]}`)
+			manifest := []byte(`{"schema_version":"ailuo.package.v3","id":"demo.pkg","version":"1.0.0","components":[{"id":"core","mode":"hosted","entrypoint":"../outside"}]}`)
 			os.WriteFile(filepath.Join(dir, "manifest.json"), manifest, 0o640)
```

golangci-lint (2.13.2)

[error] Error return value of `os.WriteFile` is not checked

(errcheck)